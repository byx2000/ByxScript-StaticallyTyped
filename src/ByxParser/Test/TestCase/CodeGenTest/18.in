function int f()
{
	int i = 300;
	return 400;
}

function void main()
{
	
}