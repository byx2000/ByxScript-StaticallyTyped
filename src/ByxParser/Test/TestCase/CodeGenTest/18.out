2
0 5
1 0

0

iconst 300
istore 0
iconst 400
ret
ret

ret